### DISCUSSION TITLE
Hook into create process for simple modal resources

### DISCUSSION THREAD

**Discussion by studiowizjo** [filamentphp/filament#4078] [Help] [answered]
Is there any way to hook into create process for simple modal resources?

I tried to use `handleRecordCreation` or `create` methods in my Resource, but they seems to be not called when submiting form.  I also inspected `Filament\Resources\Resource`, but didn`t found any trace for something usefull.

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/resources/listing-records#creating-records-in-modals

**Reply by studiowizjo** [reply]
Hmm, isn`t it for actions, not [simple modal resources](https://filamentphp.com/docs/2.x/admin/resources/getting-started#simple-modal-resources)? Using this approach I will end up with create action on each table row, but I would prefer one default button above table, on right.

**Reply by danharrin** [reply]
The Create action is inside the List page file, not the resource file. You don't need to add anything to table rows
